server/controllers/review.js

```diff
@@ -0,0 +1,70 @@
+const mongoose = require('mongoose');
+const asyncHandler = require('express-async-handler');
+
+const User = require('../models/User');
+const Profile = require('../models/Profile');
+const Review = require('../models/Review');
+
+// @route GET/reviews/:id
+// Get all reviews for a single user
+exports.searchReviews = asyncHandler(async (req,res,next) => {
```

Comment:
Looks good! Not sure about the name of this one though, I'd expect `searchReviews` to take a search query and only return matching reviews. Maybe it could be `getReviews` instead?

Reply:
Yes, changing the name to `getReviews` is more apt.
I have updated the name accordingly, thanks!